plugins/woocommerce-blocks/assets/js/base/components/cart-checkout/totals/shipping/shipping-address.tsx

```diff
@@ -19,27 +26,49 @@ export const ShippingAddress = (): JSX.Element => {
 		select( CHECKOUT_STORE_KEY ).prefersCollection()
 	);
 
+	const hasRates = hasShippingRate( shippingRates );
+
+	const { showCalculator } = useContext( ShippingCalculatorContext );
+
 	const formattedAddress = prefersCollection
 		? getPickupLocation( shippingRates )
 		: formatShippingAddress( shippingAddress );
 
+	const deliversToLabel = hasRates
+		? // Translators: <address/> is the formatted shipping address.
+		  __( 'Delivers to <strong><address/></strong>', 'woocommerce' )
```

Comment:
Sorry to nit some more but is there any reason why `strong` needs to be a separate item?

Personally I would have had the interpolated element be:

```
address: <strong>{ formattedAddress }</strong>,
```

Reply:
No good reason! Thanks I will fix that

Reply:
Addressed [Simplify usage of createInterpolateElement](https://github.com/woocommerce/woocommerce/pull/52658/commits/0f4ae5616703ee6b54943451ed8ebeac4ea2a2b1)